modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc

```diff
@@ -47,6 +47,33 @@ In the following example, the app will perform an `ADD_TASK` mutation which will
 
 For more information, see link:https://www.apollographql.com/docs/react/essentials/mutations.html#props[Apollo's document about mutations].
 
+== Offline Workflow
+
+If a mutation occurs while the device is offline, the client.mutate function:
+
+- returns immediately
+- returns a promise with an error
+
+You can check error object to isolate errors relating to offline state.
+`watchOfflineChange` method on error object will allow to watch when a change will be synced back to server,
```

Comment:
```suggestion
To sync an offline change with the server when a device comes back online, use the `watchOfflineChange()` method.
```

Reply:
This method do not sync the changes. It will watch and notify users when sync will happen.

Reply:
How does this read:
"Invoking the `watchOfflineChange()` method on an _error_ object watches for when an offline change is synced with the server, and sends an notification when triggered."